frontend/packages/topology/src/layouts/ColaLayout.ts

```diff
@@ -74,133 +108,516 @@ class ColaNode implements webcola.Node {
     this.yy = undefined;
   }
 
-  getRadius(): number {
-    const { width, height } = this.node.getBounds();
-    return Math.max(width, height) / 2;
+  get radius(): number {
+    return Math.max(this.node.getBounds().width, this.node.getBounds().height) / 2;
+  }
+
+  get collisionRadius(): number {
+    return Math.max(this.width, this.height) / 2;
```

Comment:
Why are these different?

Reply:
Width and Height are used by Cola to determine the node placement so those include the give `nodeDistance` options.  We use the `collisionRadius` for the force simulation to keep nodes that extra distance apart. We use radius only for the link distance and should not include the extra distance.

Reply:
Fair enough. It's all about the visual outcome and interaction.